src/main/java/wooteco/subway/member/dao/MemberDao.java

```diff
@@ -39,19 +39,15 @@ public Member insert(Member member) {
 
     public void update(Member member) {
         String sql = "update MEMBER set email = ?, password = ?, age = ? where id = ?";
-        jdbcTemplate.update(sql, new Object[]{member.getEmail(), member.getPassword(), member.getAge(), member.getId()});
+        jdbcTemplate.update(sql,
+            new Object[] {member.getEmail(), member.getPassword(), member.getAge(), member.getId()});
```

Comment:
```suggestion
        jdbcTemplate.update(sql,
            member.getEmail(), member.getPassword(), member.getAge(), member.getId());
```

역시 제공된 코드이지만, 메서드가 아래와 같기 때문에 배열로 감싸는 것은 불필요합니다.

```java
@Override
public int update(String sql, @Nullable Object... args) throws DataAccessException {
	return update(sql, newArgPreparedStatementSetter(args));
}
```

Reply:
제공된 메서드를 개선한다고 했는데 미처 신경쓰지 못했네요..ㅜㅜ
관련 부분 모두 수정했습니다 :)